packages/common/leaders/components/index.marko

```diff
@@ -16,10 +15,12 @@ $ const {
       <if(src)>
         <img src=src alt=alt class="leaders__logo" />
       </if>
+      <if(columns > 1)>
+        <div class="leaders__callout">
```

Comment:
Since you're flexing these,  its best to wrap text with an element, otherwise other elements that get added later may not align as expected... change to:

```html
<div>Browse these</div>
<div class="leaders__suppliers-callout>leading suppliers</div>
```

See CSS changes below regarding the suppliers callout...